### PR TITLE
Use pulpcore-manager

### PR DESCRIPTION
**Pull request by ekohl**
This requires at least Pulp 3.2.0, but since f59265ebb9fbe9cceddad37400849f8935538d1e that's been the required minimum version.

### CODE REVIEW

File: spec/classes/pulpcore_spec.rb [hunk 1]
```diff
@@ -35,9 +35,9 @@
           is_expected.to contain_class('postgresql::server')
           is_expected.to contain_postgresql__server__db('pulpcore')
           is_expected.to contain_pulpcore__admin('migrate --noinput')
-          is_expected.to contain_exec('python3-django-admin migrate --noinput')
+          is_expected.to contain_exec('pulp-manager migrate --noinput')
```
Comment (wbclark) [inline spec/classes/pulpcore_spec.rb:38]:
```suggestion
          is_expected.to contain_exec('pulpcore-manager migrate --noinput')
```

File: spec/classes/pulpcore_spec.rb [hunk 2]
```diff
@@ -35,9 +35,9 @@
...
           is_expected.to contain_pulpcore__admin('reset-admin-password --random')
-          is_expected.to contain_exec('python3-django-admin reset-admin-password --random')
+          is_expected.to contain_exec('pulp-manager reset-admin-password --random')
```
Comment (wbclark) [inline spec/classes/pulpcore_spec.rb:40]:
```suggestion
          is_expected.to contain_exec('pulpcore-manager reset-admin-password --random')
```

File: spec/defines/admin_spec.rb [hunk 3]
```diff
@@ -6,23 +6,18 @@
...
         context 'default parameters' do
           it do
             is_expected.to compile.with_all_deps
-            is_expected.to contain_exec('python3-django-admin help')
-              .with_environment([
-                'DJANGO_SETTINGS_MODULE=pulpcore.app.settings',
-                'PULP_SETTINGS=/etc/pulpcore/settings.py',
-                'PULP_STATIC_ROOT=/var/lib/pulp/assets',
-              ])
+            is_expected.to contain_exec('pulp-manager help')
```
Comment (wbclark) [inline spec/defines/admin_spec.rb:19]:
```suggestion
            is_expected.to contain_exec('pulpcore-manager help')
```

File: spec/defines/admin_spec.rb [hunk 4]
```diff
@@ -34,18 +29,13 @@
...
 
           it do
             is_expected.to compile.with_all_deps
-            is_expected.to contain_exec('python3-django-admin migrate --noinput')
-              .with_environment([
-                'DJANGO_SETTINGS_MODULE=pulpcore.app.settings',
-                'PULP_SETTINGS=/etc/pulpcore/settings.py',
-                'PULP_STATIC_ROOT=/var/lib/pulp/assets',
-              ])
+            is_expected.to contain_exec('pulp-manager migrate --noinput')
```
Comment (wbclark) [inline spec/defines/admin_spec.rb:37]:
```suggestion
            is_expected.to contain_exec('pulpcore-manager migrate --noinput')
```

File: spec/defines/admin_spec.rb [hunk 5]
```diff
@@ -60,12 +50,8 @@
             is_expected.to compile.with_all_deps
             is_expected.to contain_pulpcore__admin('help').with_pulp_settings('/etc/pulp/settings.py')
             is_expected.to contain_concat('pulpcore settings')
-            is_expected.to contain_exec('python3-django-admin help')
-              .with_environment([
-                'DJANGO_SETTINGS_MODULE=pulpcore.app.settings',
-                'PULP_SETTINGS=/etc/pulp/settings.py',
-                'PULP_STATIC_ROOT=/var/lib/pulp/assets',
-              ])
+            is_expected.to contain_exec('pulp-manager help')
```
Comment (wbclark) [inline spec/defines/admin_spec.rb:53]:
```suggestion
            is_expected.to contain_exec('pulpcore-manager help')
```

**Comment by wbclark** [comment]
On my test machine, I am getting the expected output when running `$ sudo PULP_SETTINGS=/etc/pulp/settings.py pulpcore-manager dumpdata auth.User`

I think this is good to go once tests are fixed. Thanks @ekohl ! :)

**Comment by evgeni** [comment]
Maybe? Probably better to ask the Pulp team though.

**Comment by ehelms** [comment]
Sigh, it does. Let's file an issue (and/or open a PR to add this with a default to the standard system path to make life easier for all).

**Comment by ekohl** [comment]
@bmbouter suggested we should also execute the admin command as the pulp user rather than root. I'll add that as well.

**Pull request merged**